draft-ietf-ppm-dap.md

```diff
@@ -1556,19 +1559,17 @@ reports in the batch.
 
 ##### Boundary Check
 
-For fixed-size tasks, the batch boundaries are defined by "batch IDs" selected
-by the Leader. Thus the Aggregator needs to check that the query is associated
+For fixed-size tasks, the batch boundaries are defined by consecutive numeric
+"batch IDs". Thus the Aggregator needs to check that the query is associated
 with a known batch ID:
 
-* If the query contained in the CollectReq has type `by-batch-id`, the
-  Leader checks that the provided batch ID corresponds to a batch ID it
-  returned in a previous CollectResp for the task.
-
-  [OPEN ISSUE: Consider making batch IDs predictable. See issue #301.]
+* If the query contained in the CollectReq has type `by-batch-id`, the Leader
+  MAY fail if the batch ID is considered out of range (too small or too large).
 
-* The Helper checks that the batch ID provided by the leader in its
+* The Helper checks that the batch ID provided by the Leader in its
   AggregateShareReq corresponds to a batch ID used in a previous
-  AggregateInitializeReq for the task.
+  AggregateInitializeReq for the task. The Helper MAY fail if the batch ID is
```

Comment:
```suggestion
  AggregateInitializeReq for the task. The Helper MAY abort with error "XXX" if the batch ID is
```

Reply:
Same as above: in the Batch Validation section, these boundary conditions are specified as causing a failure of type "batchInvalid". I reworded this from "failing" to "checking", since this section should be describing a set of checks.